tests/test_convert_md_to_mdx.py

```diff
@@ -129,6 +130,62 @@ def test_process_md(self):
 &amp;lt;>"""
         self.assertEqual(process_md(text, page_info), expected_conversion)
 
+    def test_convert_include(self):
+        path = Path(__file__).resolve()
+        page_info = {"path": path}
+
+        # test canonical
+        text = """<include>
+{"path": "./data/convert_include_dummy.txt",
+"start-after": "START header_1",
+"end-before": "END header_1"}
+</include>"""
```

Comment:
looks like this variable is overwritten (in line 145) without being used. Otherwise, the PR is lgtm !

Reply:
Yep exactly what I thought when I wrote it (see https://github.com/huggingface/doc-builder/pull/370#issuecomment-1528897061)

I just copied the layout of the `literalinclude` test-case, and I assume that was just left there as a reference.

Reply:
(see https://github.com/huggingface/doc-builder/blob/d563edee6ac63374dee3212b5e1a76eb7b4e426a/tests/test_convert_md_to_mdx.py#L135 for what I'm referring to)

Reply:
Shall I just replace both instances with a comment?

Reply:
> Shall I just replace both instances with a comment?

Yes, please do so. And once you push the commit, happy to merge the PR 🚀

Reply:
Done 👍